nuxt/src/assets/scss/components/_hero.scss

```diff
@@ -414,6 +414,14 @@
 
 .hero__headline {
   --color--headlines: var(--hero-copy-color);
+
+    .highlight {
```

Comment:
We do already have highlight classes for several cases. Not sure how many hero's already use the .highlight and have a different styling. This might break some of these. Maybe add another class here https://c4a8.github.io/c4a8-web-shared-components/?path=/story/docs-typography--typography instead of just in the hero.